Mathlib/Data/List/Basic.lean

```diff
@@ -1373,6 +1373,12 @@ theorem foldl_op_eq_op_foldr_assoc :
   | a :: l, a₁, a₂ => by
     simp only [foldl_cons, foldr_cons, foldl_assoc, ha.assoc]; rw [foldl_op_eq_op_foldr_assoc]
 
+theorem foldr_assoc : ∀ {l : List α} {a₁ a₂}, l.foldr op (op a₁ a₂) = op (l.foldr op a₁) a₂
```

Comment:
```suggestion
theorem foldr_left_comm : ∀ {l : List α} {a₁ a₂}, l.foldr op (op a₁ a₂) = op (l.foldr op a₁) a₂
```
no?

Reply:
This is consistent with `foldl_assoc` immediately above. Happy if these get renamed together subsequently.